virttest/libvirt_cgroup.py

```diff
@@ -134,14 +135,29 @@ def __get_standardized_cgroup1_info(self, virsh_cmd=None):
                           the blkio related cgroup info will be returned
         :return: A dict containing the cgroup info
         """
+
+        def __get_cg_file_path(cg_key, cg_path, cg_file_name):
+            """
+            Get the path to a cgroup param file
+
+            :param cg_key: The cgroup param name used in libvirt
+            :param cgroup_path: The cgroup root path
+            :param cg_file_name: The file name of the cgroup param
+            :return: The path to the cgroup param file
+            """
+            if cg_key in ["weight", "cpu_shares"] and libvirt_version.version_compare(6, 0, 0):
```

Comment:
Is it '6, 0, 0' is expected? I saw you said in the PR description that the change was introduced from libvirt 7.0.0-5.

Reply:
It was also backported in non-av version during I submitted the PR, so I udpated it to 6.0.0. 
The bug to track the non-av is https://bugzilla.redhat.com/show_bug.cgi?id=1798463

Reply:
I wonder if you can simply check whether there is 'libvirt' in the cg_path has 'libvirt' instead of checking the libvirt version.

Reply:
BTW, tests passed with your changes in my env:
_Before fix:_
` (1/3) type_specific.io-github-autotest-libvirt.guest_resource_control.control_cgroup.vm_running.hot.schedinfo.cpu_share.positive: FAIL: schedinfo checking failed. (10.23 s)`

_After:_
 ```
(1/3) type_specific.io-github-autotest-libvirt.guest_resource_control.control_cgroup.vm_running.hot.schedinfo.cpu_share.positive: PASS (9.55 s)
 (2/3) type_specific.io-github-autotest-libvirt.guest_resource_control.control_cgroup.vm_running.hot.blkiotune.weight.positive: PASS (14.21 s)
 (3/3) type_specific.io-github-autotest-libvirt.guest_resource_control.control_cgroup.vm_running.hot.blkiotune.all_in_one.positive: PASS (14.04 s)


```